contracts/actions/convex/ConvexClaim.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+
+import "../../interfaces/convex/IBooster.sol";
+import "../../interfaces/convex/IBaseRewardPool.sol";
+import "../../interfaces/convex/IRewardPool.sol";
+import "../../utils/TokenUtils.sol";
+import "./helpers/ConvexHelper.sol";
+import "../ActionBase.sol";
+
+contract ConvexClaim is ConvexHelper, ActionBase {
+    using TokenUtils for address;
+
+    /// @param from address for which to claim rewards
+    /// @param to address that will receive the rewards
+    struct Params {
+        address from;
+        address to;
+        address rewardContract;
```

Comment:
Is this something that will change or can we hardcode it?

Reply:
rewardContract differs from pool to pool